src/keys.c

```diff
@@ -1,4 +1,4 @@
-/* keys.c
+/* KEYS.c
```

Comment:
The lower case "keys" in a comment was a problem?

Reply:
oh dear, I don't even have a working hypothesis of how that happened.  will push fix for it in due course.